tests/unit/test_parser.py

```diff
@@ -20,43 +23,39 @@ def test_parser_multiple():
         "away_team",
     ]
     print("FULL MODEL TEST - RUNNING TEST MODEL...")
-    for line in input_str.split("\n"):
-        line = line.lstrip().rstrip()
-        if not line:
-            continue
-        print(f"----running for line [{line}]")
-        print(line)
-        print("{", [(x.__class__.__name__, x.value) for x in scanner(line)], "}")
-        print(Parser(scanner(line), data_names, line).statement())
+    scanned_lines = Scanner(input_str).scan()
+    for line in scanned_lines:
+        print(Parser(line, data_names, line).statement())
```

Comment:
I don't think there's anything really wrong with print statements in here, but pytest eats them by default so they won't be terribly useful unless you're running pytest with prints turned on.